templates/cluster-template-clusterclass.yaml

```diff
@@ -288,7 +288,7 @@ spec:
 apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
 kind: KubeadmConfigTemplate
 metadata:
-  name: ${CLUSTER_CLASS_NAME}-kcfg-0
+  name: nutanix-quick-start-kcfg-0
```

Comment:
nit: given we have a e2e test set named `quick-start` perhaps we can use a different default name? just calling it `clusterclass-default` would perhaps be a bit more obvious if/when we encounter it as a stale resource in our dev/CI environments.